### PR TITLE
Adding more information to the installation docs

### PR DESCRIPTION
**Pull request by mikeethedude**
## Purpose

There is a need to have more thorough documentation for setting up Sous. I know there will be a need with 3.x releasing soon. This adds a bit more information about the requirements and gives some additional information for the uninitiated.

### CODE REVIEW

**Comment by mikeethedude** [comment]
We have information on Semantic Versioning in the README.md here, but should it be here or in the new docs being created? It's more information that has to do with contributing or releasing the project. It may do better to have that in the Sous Notion page maybe?

File: README.md
Comment (ccjjmartin) [inline README.md:15]: @mikeethedude In reality we should be using Node 16 because that is what the latest version of Emulsify will require along with the Emulsify CLI.

File: README.md
Comment (ccjjmartin) [inline README.md:16]: @mikeethedude If we are specifying versions we might want to go ahead and specify version 2 here.

File: README.md
Comment (ccjjmartin) [inline README.md:14]: @mikeethedude I agree with this, so no change necessary but flagging that 7.4 is technically security upgrades only now: https://www.php.net/supported-versions.php

File: README.md
Comment (ccjjmartin) [inline README.md:47]: @mikeethedude Actually this whole section isn't an officially recommended method of setting up the theme with Sous.  When you composer create-project the project it runs the post composer require command: https://github.com/fourkitchens/sous-drupal-project/blob/2.x/composer.json#L108

Which is this script here: https://github.com/fourkitchens/sous-drupal-project/blob/2.x/sous/Starter.php#L19

So until we update this documentation to reflect using the new Emulsify CLI (my rock for next quarter), we should probably leave this one to reflect the composer create-project scripts or if you wanted to run it manually (after a clone) you can do a `lando composer post-create-project-cmd` and it will run the script.

File: README.md
Comment (ccjjmartin) [inline README.md:63]: @mikeethedude Drupal console isn't working as of Drupal core 9.2+.  So I am not actually sure we should recommend using it as Sous uses core 9.2+ now.  I don't have a good solution for this though ... and I thought this might have already been in the docs somewhere ...

**Review by ccjjmartin — COMMENTED** [review]
@mikeethedude I took a look at this one.

File: README.md
Comment (mikeethedude) [reply README.md:63]: It was in the docs already. I moved it down. I'll just toss it out

File: README.md
Comment (mikeethedude) [reply README.md:15]: True. I was going off of the nvm setting which is lts/erbium or 12.13.0

File: README.md
Comment (mikeethedude) [reply README.md:47]: Interesting! I didn't catch that. I'll update this too.

File: README.md
Comment (mikeethedude) [reply README.md]: Good to note. We'll want to update everyone soon here. 8.x here we go.

**Comment by mikeethedude** [comment]
@ccjjmartin I've updated based on your feedback.

**Review by ccjjmartin — APPROVED** [review]
@mikeethedude :shipit: 

I can't answer the question as to where it is better for the semantic version docs to live.  But I do know it kind of reminds people to be aware of it when they are contributing (think about where you are merging etc).

Speaking of, make sure you merge this into the release/3x branch after you merge it into 2.x 🚢

**Review by callinmullaney — APPROVED** [review]

**Pull request merged**
